vendir.yml

```diff
@@ -20,10 +20,13 @@ directories:
           path: vendor/cilium/install/kubernetes
         ignorePaths:
           - cilium/Chart.yaml
+          - cilium/Chart.lock
           - cilium/files/policies/*
           - cilium/templates/default-policies/*
           - cilium/templates/extra-policies/*
           - cilium/templates/**/networkpolicy.yaml
           - cilium/templates/**/podsecuritypolicy.yaml
-          - cilium/templates/cilium-cni-configmap.yaml
-
+          - cilium/charts/cilium-giantswarm-cni-customconf/*.yaml
```

Comment:
couldn't we use wildcards here? I mean something like `cilium/charts/**/*`?

Reply:
Unfortunately not :(

It gives me this error:

```
vendir: Error: Syncing directory 'helm':
  Copying existing content to staging 'helm':
    Moving source file 'helm/cilium/charts/cilium-giantswarm-cni-customconf' to staging location '.vendir-tmp-3994942219/staging/cilium/charts/cilium-giantswarm-cni-customconf': helm/cilium/charts/cilium-giantswarm-cni-customconf is not a regular file
```